applications/ContactStructuralMechanicsApplication/custom_conditions/penalty_frictional_mortar_contact_axisym_condition.h

```diff
@@ -139,33 +156,46 @@ class KRATOS_API(CONTACT_STRUCTURAL_MECHANICS_APPLICATION) PenaltyMethodFriction
     ///@{
 
     /**
-     * Creates a new element pointer from an arry of nodes
+     * @brief Creates a new element pointer from an arry of nodes
      * @param NewId The ID of the new element
      * @param rThisNodes The nodes of the new element
      * @param pProperties The properties assigned to the new element
      * @return a Pointer to the new element
      */
-
     Condition::Pointer Create(
         IndexType NewId,
         NodesArrayType const& rThisNodes,
         PropertiesPointerType pProperties
         ) const override;
 
     /**
-     * Creates a new element pointer from an existing geometry
+     * @brief Creates a new element pointer from an existing geometry
      * @param NewId The ID of the new element
      * @param pGeom The  geometry taken to create the condition
      * @param pProperties The properties assigned to the new element
      * @return a Pointer to the new element
      */
-
     Condition::Pointer Create(
         IndexType NewId,
         GeometryPointerType pGeom,
         PropertiesPointerType pProperties
         ) const override;
 
+    /**
+     * @brief Creates a new element pointer from an existing geometry
+     * @param NewId the ID of the new element
+     * @param pGeom the  geometry taken to create the condition
+     * @param pProperties the properties assigned to the new element
+     * @param pMasterGeom the paired geometry
+     * @return a Pointer to the new element
+     */
+    Condition::Pointer Create(
+        IndexType NewId,
+        GeometryPointerType pGeom,
+        PropertiesPointerType pProperties,
+        GeometryPointerType pMasterGeom
+        ) const override;
```

Comment:
I thought that here you can pass both geometry as one.

Reply:
This PR is a transition, the actual geometry is used in #5394